### PR TITLE
Add new T&Cs for GDPR retention policies

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Changes proposed in this pull request

Add this section:

image

### Guidance to review

https://get-school-experience-review-pr-8.test.teacherservices.cloud/terms_and_conditions

Scroll down to 'Data storage, retention and destruction'

### CODE REVIEW

File: app/views/pages/terms_and_conditions.html.erb [hunk 1]
```diff
@@ -161,18 +161,29 @@
         <h3 class="govuk-heading-s">Accepting applications through the Get school experience service</h3>
             <p>
                 By accepting school experience applications through the Service, you agree that:
-            <p>
-            <p>
-                Once you receive an application from a school experience candidate, it's your responsibility to adhere to General Data Protection Regulation (GDPR) requirements.
             </p>
                 <ul class="govuk-list govuk-list--bullet">
+                    <li>Once you receive an application from a school experience candidate, it's your responsibility to adhere to General Data Protection Regulation (GDPR) requirements.</li>
```
Comment (gemmadallmandfe) [inline app/views/pages/terms_and_conditions.html.erb:166]:
```suggestion
                    <li>once you receive an application from a school experience candidate, it's your responsibility to adhere to General Data Protection Regulation (GDPR) requirements</li>
```

File: app/views/pages/terms_and_conditions.html.erb [hunk 2]
```diff
@@ -161,18 +161,29 @@
...
                 </ul>
             <p>
                 As a school, it's your responsibility to decide if a school experience candidate complies with your DBS and safeguarding policies.
             </p>
+        <h3 class="govuk-heading-s">Data storage, retention and destruction</h3>
+            <p>
+                You must store shared data securely and have appropriate safeguards in place to protect data against unlawful or unauthorised processing. Personal data must be kept only as long as needed to carry out activities relating to administering school experience.
+            </p>
+            <p>
+                You must set an appropriate time limit, in line with the General Data Protection Regulation, for retaining data before erasure or review.
```
Comment (gemmadallmandfe) [inline app/views/pages/terms_and_conditions.html.erb:179]:
```suggestion
                You must set an appropriate time limit, in line with GDPR requirements, for retaining data before erasure or review.
```

given that we've defined GDPR in the section before, should we shorten it here?

File: app/views/pages/terms_and_conditions.html.erb [hunk 3]
```diff
@@ -161,18 +161,29 @@
...
+            </p>
+            <p>
+                You should destroy the data when it is no longer needed, or when the retention schedule has expired.
+            </p>
+            <p>
+                You should follow the NCSC guidance for secure sanitisation.
```
Comment (gemmadallmandfe) [inline app/views/pages/terms_and_conditions.html.erb:185]:
```suggestion
                You should follow the National Cyber Security Centre (NCSC) guidance for secure sanitisation.
```
think we need to put this in full as it is not defined elsewhere on the page

we _could_ include a link to the guidance (do Manage?)

**Review by gemmadallmandfe — COMMENTED** [review]
@MylesJarvis Some minor suggestions!

File: app/views/pages/terms_and_conditions.html.erb (hunk 3 above)
Comment (MylesJarvis) [reply app/views/pages/terms_and_conditions.html.erb:185]: Manage do not, I propose we leave it for now.

**Pull request merged**